### ISSUE TITLE
How to pass data between transitions

### ISSUE THREAD

**Issue by alxmath** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2612]
I'm developing a app with transitions. Is there some way to pass data between transitions? I'm using UserControl as slides.

**Comment by Erapchu** [comment]
Yes, it's possible. You can make your own logic to pass data between transitions. For example call some configure method on click Next. Data object may be always in root view model, change it on next/back

**Comment by MichelMichels** [comment]
@alxmath This library is design-focused, so logic related questions are almost always out-of-scope. As @Erapchu says, you'll have to program this yourself as there's no builtin support for this.

For examples on how to achieve this, you could look at the repository of @Keboo at https://github.com/Keboo/MaterialDesignInXaml.Examples/tree/master/Transitioner
